src/cbl-mariner/2.0/crossdeps-builder/Dockerfile

```diff
@@ -26,6 +26,11 @@ RUN wget https://packages.microsoft.com/cbl-mariner/2.0/prod/extended/x86_64/con
     tdnf install -y \
         libbsd-devel
 
+ENV NODE_VERSION 20.11.1
+RUN mkdir /usr/share/node && \
+    curl -sSL https://nodejs.org/dist/v${NODE_VERSION}/node-v${NODE_VERSION}-linux-x64.tar.xz | tar xf - -J -C /usr/share/node --strip-components=1 && \
```

Comment:
please add checking the sha like we do for llvm-project.src.tar.xz below

Reply:
The process to check the downloaded file is https://github.com/nodejs/node?tab=readme-ov-file#verifying-binaries

Unfortunately, the key import part does not work on mariner right now. See https://github.com/microsoft/azurelinux/issues/3142. There is a [PR open](https://github.com/microsoft/azurelinux/pull/8349) to resolve this, but the change (install gnutls-devel) doesn't appear make a difference. Leaving the verification of SHA256SUM.txt for later then,